spec/rails_helper.rb

```diff
@@ -28,7 +28,6 @@
   config.file_fixture_path = "#{::Rails.root}/spec/fixtures/files"
 
   config.use_transactional_fixtures = true
-  config.global_fixtures = :users
```

Comment:
Yay! Not having these user fixtures loaded for all tests will make it a lot easier to manage users for different subdomains :-)